frontend/termbox/main.go

```diff
@@ -183,26 +189,23 @@ func (t *tbfe) renderView(v *backend.View, lay layout) {
 	x, y := sx, sy
 	ex, ey := sx+w, sy+h
 
-	style, _ := v.Settings().Get("caret_style", "underline").(string)
-	inverse, _ := v.Settings().Get("inverse_caret_state", false).(bool)
+	style := v.Settings().Get("caret_style", "underline").(string)
```

Comment:
This change will cause a panic if the setting isn't of type `string`. See http://golang.org/ref/spec#Type_assertions